ui/components/app/modals/import-token-exit-modal/import-token-exit-modal.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import * as actions from '../../../../store/actions';
+import { DEFAULT_ROUTE } from '../../../../helpers/constants/routes';
+import {
+  Text,
+  ButtonSecondary,
+  ButtonPrimary,
+  Box,
+  Modal,
+  ModalContent,
+  ModalOverlay,
+} from '../../../component-library';
+import {
+  TextAlign,
+  FontWeight,
+  TextVariant,
+  Size,
+  Display,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+// export default function ImportTokensExitModal() {
+export const ImportTokensExitModal = ({ onClose }) => {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  return (
+    <Modal isOpen>
+      <ModalOverlay />
+      <ModalContent className="top-class">
```

Comment:
This CSS className needs to be much more specific.  Maybe `import-token-exit-token-confirmation__content`?